src/app/lib/config/services/ws.ts

```diff
@@ -411,32 +433,44 @@ export const service: DefaultServiceConfig = {
             id: 'burmese',
             href: 'https://www.bbc.com/burmese',
             label: 'မြန်မာသတင်းများ',
+            lang: 'my',
+            latinTransliteration: 'Myanmar satinmyar',
           },
           {
             id: 'zhongwen',
             href: 'https://www.bbc.com/zhongwen/simp',
```

Comment:
Let's update this - it needs to be the default variant
```suggestion
            href: 'https://www.bbc.com/zhongwen/trad',
```